docs/testing/troubleshoot-manifest.md

```diff
@@ -67,14 +67,14 @@ You can use runtime logging to debug your add-in's manifest as well as several i
 
 1. Make sure that you are running Office 2016 desktop build **16.0.7019** or later. 
 
-2. Add the `RuntimeLogging` registry key under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\WEF\Developer\`. 
+2. Add the `RuntimeLogging` registry key under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\Wef\Developer\`. 
```

Comment:
Seems like "WEF" should remain capitalized -- it's showing like that for me, anyway:

![image](https://user-images.githubusercontent.com/3019529/67607865-d1b56280-f73a-11e9-9be9-59b1f672cba3.png)